tests/functional/tests/rgw/test_rgws.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+
+class TestRGWs(object):
+
+    @pytest.mark.no_docker
+    def test_ceph_rgw_package_is_installed(self, node, Package):
+        assert Package("radosgw").is_installed
+
+
+    def test_rgw_services_are_running(self, node, Service):
+        # TODO: figure out way to paramaterize node['rgws'] for this test
+        for rgw in node["rgws"]:
+            assert Service("ceph-radosgw@%s" % rgw).is_running
```

Comment:
This is what I use for testing mons: https://github.com/ceph/ceph-ansible/blob/master/tests/functional/tests/mon/test_mons.py#L14-L16

``node['vars']`` already contains the output of ``Ansible.get_variables()``